packages/query-core/src/__tests__/queryClient.test.tsx

```diff
@@ -400,6 +400,27 @@ describe('queryClient', () => {
     })
   })
 
+  describe('isFetching', () => {
+    test('should return length of fetching queries', async () => {
+      const key1 = queryKey()
+      const key2 = queryKey()
+
+      expect(queryClient.isFetching()).toBe(0)
+      queryClient.prefetchQuery({
+        queryKey: key1,
+        queryFn: () => sleep(300).then(() => 'data'),
+      })
+      expect(queryClient.isFetching()).toBe(1)
+      queryClient.prefetchQuery({
+        queryKey: key2,
+        queryFn: () => sleep(200).then(() => 'data'),
```

Comment:
since we use real timers per default, please keep those times short (5ms and 10ms at most), or use fake timers and advance them how you want.